monkey/agent_plugins/exploiters/powershell/config-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+    "properties": {
+        "target_ports": {
+            "title": "Target ports",
+            "description": "A list of ports that the PowerShell exploiter will try to exploit.",
+            "type": "array",
+            "uniqueItems": true,
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 65535
+            },
+            "default": [
+                5985,
+                5986
+            ]
+        },
+        "try_all_discovered_open_ports": {
+            "title": "Try all discovered open ports",
+            "description": "Attempt to exploit PowerShell on all open ports with unknown services discovered from network scanning.",
+            "type": "boolean",
+            "default": false
+        },
```

Comment:
This option has the essentially the same issue as stated above. I think we can drop this option.